GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAndroidResource.java

```diff
@@ -313,6 +313,23 @@ public synchronized final InputStream getStream() throws IOException{
         return stream;
     }
 
+    /**
```

Comment:
The API itself seems confusing since for one like @liaxim mentions, it breaks the clean encapsulation provided by GVRAndroidResource. What happens when I create a NETWORK type resource and then use the setStream call? Does it still remain a GVRAndroidResource of type NETWORK or does it become something else? 

I guess we might get a better idea if we see an example of this API in use. To me we could still work with the constructor where a GVRAndroidResource created using the input stream can be provided to the assetloader to be loaded.